config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users, only: [:index,:show] do
+    resources :posts, only: [:index,:show,:new, :create]
```

Comment:
- [ ] Let's avoid exposing unnecessary routes. From the project guideline, you should only have 4 routes. Kindly remove the `new` and `create` routes for the posts.

- [ ] Kindly remove the empty post new view file located here: `app/views/posts/new.html.erb`.